tests/ArcCommander.Tests.NetCore/ArcCommander/AssayTests.fs

```diff
@@ -213,15 +276,19 @@ let testAssayRegister =
             let assayOption = API.Assay.tryGetByFileName assayFileName study.Assays.Value
             Expect.isSome assayOption "New Assay was not added to study"
             Expect.equal assayOption.Value testAssay "New Assay is missing values"
-            Expect.equal study.Assays.Value.Length 2 "Assay missing"
+            Expect.equal study.Assays.Value.Length 1 "Assay missing"
         )
-        testCase "StudyNameNotGivenUseAssayNameNoDuplicateAssay" (fun () ->             
+        testCase "StudyNameNotGivenUseAssayNameNoDuplicateAssay" (fun () ->
```

Comment:
Same issue as with "doNothingIfAssayExisting" test. As now an assay with the given name was not previously added, the case that should have been tested is now not possible to occur.